src/Uno.UI.RemoteControl/HotReload/ClientHotReloadProcessor.Xaml.cs

```diff
@@ -102,14 +102,14 @@ bool IsSameBaseUri(FrameworkElement i)
 					{
 #if __IOS__
 						case UserControl userControl:
-							if (XamlReader.LoadUsingXClass(fileContent) is UIKit.UIView newInstance)
+							if (XamlReader.LoadUsingXClass(fileContent, uri.ToString()) is UIKit.UIView newInstance)
```

Comment:
Just to make sure we don't have escaping issues.

```suggestion
							if (XamlReader.LoadUsingXClass(fileContent, uri.OriginalString) is UIKit.UIView newInstance)
```